index.html

```diff
@@ -1420,183 +1419,189 @@ <h5>
-        <p data-lang="zh-hans" class="checkme">标号相对于点号，有标示词组或语句特定性质的作用。包含引号、括号（夹注号）、破折号、删节号（省略号）、着重号、连接号、间隔号、书名号、专名号、分隔号。</p>
-        <p data-lang="zh-hant">標號相對於點號，有標示詞組或語句之特定性質的作用。包含引号、括号（夾注號）、破折号、刪節號（省略号）、着重号、连接号、间隔号、书名号、专名号、分隔号。</p>
+        <p data-lang="zh-hans">标号相对于点号，有标示词组或语句特定性质的作用。包含引号、括号（夹注号）、破折号、省略号（删节号）、着重号、连接号、间隔号、书名号、专名号、分隔号。</p>
+        <p data-lang="zh-hant">標號相對於點號，有標示詞組或語句特定性質的作用。包含引號、括號（夾注號）、破折號、刪節號（省略號）、著重號、連接號、間隔號、書名號、專名號、分隔號。</p>
         <ol>
           <li>
-            <p data-lang="en"><span class="leadin">Quotation Marks/Brackets</span></p>
-            <p data-lang="zh-hans" class="checkme"><span class="leadin">引号</span></p>
+            <p data-lang="en"><span class="leadin">Quotation Marks</span></p>
+            <p data-lang="zh-hans"><span class="leadin">引号</span></p>
             <p data-lang="zh-hant"><span class="leadin">引號</span></p>
             <p data-lang="en">Quotation marks, usually used in pairs, are commonly used to emphasize certain characters or words, or to indicate the beginning and ending of the dialog or quoted content. If there is a need to use a bracket within a pair of brackets, the shape of the inner quotation marks will differ from the parent quotation marks. Quotation marks are a kind of bracket.</p>
-            <p data-lang="zh-hans" class="checkme">引号用于强调字词，或作为引用话语、文献的起讫边界。引号中再次使用引号时，使用内引号。外引号与内引号有字形上的差异以便识别。引号属于夾注符号。</p>
+            <p data-lang="zh-hans">引号用于强调字词，或作为引用话语、文献的起讫边界。引号中再次使用引号时，使用内引号。外引号与内引号有字形上的差异以便识别。引号属于夾注符号。</p>
             <p data-lang="zh-hant">引號用於強調字詞，或作為引用話語、文獻的起訖邊界。引號中再次使用引號時，使用內引號。外引號與內引號有字形上的差異以便識別。引號屬於夾注符號。</p>
             <p data-lang="en">When there is a need for quotation marks, Taiwan will apply single quotation marks first, followed by double quotation marks. Single quotation marks include <span class="uname">U+300C LEFT CORNER BRACKET</span> [「] and <span class="uname">U+300D RIGHT CORNER BRACKET</span> [」]; double quotation marks include <span class="uname">U+300E LEFT WHITE CORNER BRACKET</span> [『] and <span class="uname">U+300F RIGHT WHITE CORNER BRACKET</span> [』].</p>
-            <p data-lang="zh-hans" class="checkme">在台湾，采用先单、后双的引号体例。单引号，包含开始单直角引号<span class="uname" translate="no">U+300C LEFT CORNER BRACKET</span> [「]与结束单直角引号 <span class="uname" translate="no">U+300D RIGHT CORNER BRACKET</span> [」];双引号，包含开始双 直角引号 <span class="uname" translate="no">U+300E LEFT WHITE CORNER BRACKET</span> [『]与结束双直角引号 <span class="uname" translate="no">U+300F RIGHT WHITE CORNER BRACKET</span> [』]。</p>
+            <p data-lang="zh-hans">在台湾，采用先单、后双的引号体例。单引号，包含开始单直角引号<span class="uname">U+300C LEFT CORNER BRACKET</span> [「]与结束单直角引号<span class="uname">U+300D RIGHT CORNER BRACKET</span> [」]；双引号，包含开始双直角引号<span class="uname">U+300E LEFT WHITE CORNER BRACKET</span> [『]与结束双直角引号<span class="uname">U+300F RIGHT WHITE CORNER BRACKET</span> [』]。</p>
```

Comment:
Not sure why translate="no" was removed here.

Reply:
I wasn't so sure about the usage of `translate="no"` so I took down most of them. Looks like they weren't redundant. I’ll add them back.

Reply:
Using HTML's translate attribute
https://www.w3.org/International/questions/qa-translate-flag

Reply:
Thanks!